content/blog/2018/01/2018-01-05-gsoc2018-call-for-mentors.adoc

```diff
@@ -0,0 +1,92 @@
+---
+:layout: post
+:title: "Google Summer Of Code 2018: Call for mentors"
+:tags:
+- gsoc
+- gsoc2018
+- events
+- community
+:author: oleg-nenashev
+---
+
+This year we are interested to participate in
+link:https://developers.google.com/open-source/gsoc/[Google Summer of Code (GSoC)].
+As in 2016/2017, we are looking for mentors.
+So yes, we are looking for you :)
+
+image:/images/gsoc/jenkins-gsoc-logo_small.png[Jenkins GSoC, role=center, float=right]
+
+== What is GSoC?
+
+link:https://developers.google.com/open-source/gsoc/[Google Summer of Code]
+(GSoC) is an annual, international, program which encourages
+college-aged students to participate with open source projects during the summer
+break between classes.
+
+Students accepted into the program receive a stipend,
+paid by Google, to work well-defined projects to improve or enhance the Jenkins
+project.
+In exchange, numerous Jenkins community members volunteer as "mentors"
+for students to help integrate them into the open source community and succeed
+in completing their summer projects.
+
+== What do you get?
+
+* A student who works within the area of your interest on full-time for several months
+* Joint projects with Jenkins experts, lots of fun and ability to study something together
+* Limited edition of swags from Google and Jenkins project
+* Maybe: Participation in GSoC Mentor Summit and other GSoC events/meetups
+
+== Conditions
+
+Mentors are expected to...
+
+* be passionate about Jenkins
+* lead the project in the area of their interest
+* actively participate in the project during student selection, community bonding and coding phases (March - August)
+* work in teams of 2+ mentors per 1 each student
+* dedicate a consistent and significant amount of time, especially during the coding phase (_~5 hours_ per week in a team of two mentors)
+
```

Comment:
Let's use caps at the start of each of these elements, it's consistent with the previous list and IMO the preferred style.